src/Core/Logging/Extensions/ILogServiceExtensions.Internal.cs

```diff
@@ -0,0 +1,268 @@
+                formatter,
+                methodName,
+                extensionMethodName);
+        }
+
+        /// <summary>
+        /// Wrapper for <see cref="ILogServiceBase.Log{TState}(LogLevel, TState, Exception, Func{TState, Exception, string}, string)"./>
+        /// </summary>
+        private static void Internal(
+            this ILogService service,
+            Exception ex,
+            Func<object, Exception, string> formatter,
+            string methodName = "",
+            [CallerMemberName] string extensionMethodName = "")
+        {
+            service.Internal(
+                DateTimeOffset.Now,
+                null,
+                ex,
+                formatter,
+                methodName,
+                extensionMethodName);
+        }
+
+
+
+        /// <summary>
+        /// Wrapper for <see cref="ILogServiceBase.Log{TState}(DateTimeOffset, LogLevel, TState, Exception, Func{TState, Exception, string}, string)"./>
+        /// </summary>
+        private static void Internal<TState>(
+            this ILogService service,
+            DateTimeOffset timestamp,
+            TState message,
+            Func<TState, Exception, string> formatter,
+            string methodName = "",
+            [CallerMemberName] string extensionMethodName = "")
+        {
+            service.Internal(
+                timestamp,
+                message,
+                null,
+                formatter,
+                methodName,
+                extensionMethodName);
+        }
+
+        /// <summary>
+        /// Wrapper for <see cref="ILogServiceBase.Log{TState}(DateTimeOffset, LogLevel, TState, Exception, Func{TState, Exception, string}, string)"./>
+        /// </summary>
+        private static void Internal(
+            this ILogService service,
+            DateTimeOffset timestamp,
+            Exception ex,
+            Func<object, Exception, string> formatter,
+            string methodName = "",
+            [CallerMemberName] string extensionMethodName = "")
+        {
+            service.Internal(
+                timestamp,
+                null,
+                ex,
+                formatter,
+                methodName,
+                extensionMethodName);
+        }
+
+        /// <summary>
+        /// Wrapper for <see cref="ILogServiceBase.Log{TState}(DateTimeOffset, LogLevel, TState, Exception, Func{TState, Exception, string}, string)"./>
+        /// </summary>
+        private static void Internal<TState>(
+            this ILogService service,
+            DateTimeOffset timestamp,
+            TState message,
+            Exception ex,
+            Func<TState, Exception, string> formatter,
+            string methodName = "",
+            [CallerMemberName] string extensionMethodName = "")
+        {
+            Argument.NotNull(
+                service,
+                nameof(service));
+
+            if (!TryParseLogLevel(extensionMethodName, out var logLevel))
```

Comment:
Codacy found an issue: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5762132)